packages/core/src/tegel-light/components/tl-tooltip/tl-tooltip.scss

```diff
@@ -0,0 +1,58 @@
+.tl-tooltip {
```

Comment:
I see that we in other components (for example tl-button.scss) import the vars file. But not here.
Is that needed? If so we should add it :)